core-web/libs/edit-content/src/lib/fields/dot-edit-content-binary-field/dot-edit-content-binary-field.component.ts

```diff
@@ -380,9 +383,9 @@ export class DotEditContentBinaryFieldComponent
     private parseCustomMonacoOptions(
         fieldVariables: DotCMSContentTypeFieldVariable[]
     ): Record<string, string> {
-        const { monacoOptions } = getFieldVariablesParsed<{ monacoOptions: string }>(
-            fieldVariables
-        );
+        const { monacoOptions } = getFieldVariablesParsed<{
+            monacoOptions: string;
+        }>(fieldVariables);
```

Comment:
We're having prettier issues again... not sure what's going on, but there is something going on.

Reply:
Verify if you have the prettier file at the root of `core-web`